deploy/nl/catalog.yaml

```diff
@@ -59,43 +59,43 @@ indexes:
     healthcheck_query: "Life expectancy"
```

Comment:
What about the path for medium_ft?  Wonder when we can get rid of it?

Reply:
medium_ft does not have a path now or does not worth to update in anyway.

Now that the embeddings pipeline are in shape, we should plan migrate custom DC to vertex AI based model as well.